vector/src/main/java/im/vector/app/features/home/ShortcutCreator.kt

```diff
@@ -83,11 +87,7 @@ class ShortcutCreator @Inject constructor(
 
     private fun Bitmap.toProfileImageIcon(): IconCompat {
         return if (useAdaptiveIcon) {
-            val insetBmp = Bitmap.createBitmap(adaptiveIconSize, adaptiveIconSize, Bitmap.Config.ARGB_8888)
```

Comment:
this was the cause of our extra allocations, the fix was to _transform_ the bitmap transformation into a cachable glide transformation https://github.com/vector-im/element-android/pull/5276/files#diff-1f9afdb46350b185b8e9cfc6698b2885bb19fb15d98c1c65dfedfe5371615d81R173